cypress-io/cypress
# misc: improve debug loading text wrap responsiveness

- Closes https://github.com/cypress-io/cypress/issues/25667

### Additional details
Improves responsiveness of the Debug Page empty state header and remove full width header to align with Figma

### Steps to test
Run: https://github.com/cypress-io/cypress/blob/13dbd418a25a8e7b3d65ececbbd3c41dac220b33/packages/app/src/debug/empty/DebugEmptyStates.cy.tsx#L24

and check out the percy snaphots

### How has the user experience changed?
![Screen Shot 2023-02-03 at 12 11 04 PM](https://user-images.githubusercontent.com/25158820/216676470-e93ddf4c-9d1e-4e7d-acee-baffe6068caf.png)


### PR Tasks


- [na] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [na] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
